kolibri/core/assets/src/views/CoreBase/ScrollingHeader.vue

```diff
@@ -86,6 +96,11 @@
     },
     methods: {
       handleNewScrollPosition(newVal, oldVal) {
+        if (this.skipNextUpdate) {
+          this.resetSkipNextUpdateDebounced();
+          return;
+        }
```

Comment:
Sometimes `CoreBase` scrolling can get set multiple times during the course of single navigation event (especially in the case of a smooth transition, even without it).  This makes sure that the `ScrollingHeader` doesn't auto-calculate when its `isHidden` status has been set manually.  the `skipNextUpdate` property is only reset when there haven't been any scroll position changes for 500 ms.